lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -167,6 +176,69 @@ class CategoryRenderer {
     element.appendChild(passedElem);
     return element;
   }
+
+  _renderAuditGroup(audits, tag) {
+      const auditGroupElem = this._dom.createElement('details', 'lh-audit-group');
+      const auditGroupHeader = this._dom.createElement('div', 'lh-audit-group__header');
+      auditGroupHeader.textContent = tag.title;
+
+      const auditGroupDescription = this._dom.createElement('div', 'lh-audit-group__description');
```

Comment:
I saw that but wasn't convinced we wanted to reuse those, `lh-score` gives me the idea that it's a scored thing specific to audit or category when these are just display groupings. If we want to reuse the same CSS, maybe we can extract it to a different distinct reusable component without tying the category/audit dependency?